repository/src/main/resources/alfresco/subsystems/Authentication/identity-service/identity-service-authentication.properties

```diff
@@ -11,3 +11,4 @@ identity-service.realm=alfresco
 identity-service.resource=alfresco
 identity-service.credentials.secret=
 identity-service.public-client=true
+identity-service.admin-console.redirect-path=/alfresco/s/admin/admin-communitysummary
```

Comment:
The enterprise summary is located in a different URI, are users expected to change this property to deal with that? To e.g.: `alfresco/s/enterprise/admin/admin-systemsummary`?

Also, do we really need to declare property or can it always be safely calculated/reverse-engineered via code _(similarly to what is done in AOS, e.g.: [here](https://github.com/Alfresco/alfresco-aos-module/blob/support/SP/2.0.N/alfresco-aos-repo-binding/src/main/java/org/alfresco/module/aosmodule/auth/service/AosAuthenticationUriService.java#L34-L47), maybe with some additional logic to distinguish between whether the current installation of ACS is Community/Enterprise to redirect to the appropriate page)_?

Reply:
We also need to set the default redirect path for the enterprise repository. See: https://github.com/Alfresco/alfresco-enterprise-repo/compare/master...feature/ACS-6603_implement_oidc_compliance#diff-d05b5893ad9eb54d7af8dd9cf364357e81b0eb022cf2b7300dac3dddc73c20a2R130. 
We agreed with @jamalkm to make it configurable rather than hardcoded/calculated/reverse-engineered via code just in case of a potential mistake or other need. For example to change the default page after login.